### PR TITLE
feat(bake/manifests): Rudimentary bake manifest support

### PR DESCRIPTION
**Pull request by lwander**
Still missing:

1. Downloading helm values files
2. Setting helm --set values
3. Deleting staged helm charts
4. Creating non base64 artifacts
5. Polling of running bake tasks -- the bake is currently synchronous

### CODE REVIEW

**Comment by lwander** [comment]
https://github.com/spinnaker/spinnaker/issues/2636

File: rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/services/ServiceConfig.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package com.netflix.spinnaker.rosco.services;
```
Comment (ezimanyi) [inline rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/services/ServiceConfig.java]: Do we need the standard copyright text at the top of this and other new classes?

Comment (lwander) [reply rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/services/ServiceConfig.java]: Good catch, looks like my intellij hasn't been configured for this project yet

**Comment by lwander** [comment]
It's not really backwards compatible, at least from a url path perspective. 

The prior url paths were 

`/api/v1/{region}/bake`

Manifests don't have any region associated with them -- sticking a placeholder value in there is hacky and registering another path that uses a different `/api/v1/<something else>...` has potential to collide with the original path in the future.

Ideally, this probably should have been registered as

`/api/v1/bake/{region}` or something similar, giving other endpoints the chance to register something other than `{region}` as the first path token after `/api/v1`. 

Starting with `v2`, we have `/api/v2/manifest/bake`, and can add `/api/v2/image/bake/region/{region}`

**Pull request merged**